In file included from ../../components/page_load_metrics/browser/observers/ad_metrics/aggregate_frame_data.cc:5:
../../components/page_load_metrics/browser/observers/ad_metrics/aggregate_frame_data.h:42:8: error: no template named 'optional' in namespace 'std'
  std::optional<base::TimeDelta> first_ad_fcp_after_main_nav_start() const {
  ~~~~~^
../../components/page_load_metrics/browser/observers/ad_metrics/aggregate_frame_data.h:132:8: error: no template named 'optional' in namespace 'std'
  std::optional<base::TimeDelta> first_ad_fcp_after_main_nav_start_;
  ~~~~~^
2 errors generated.

--- a/components/page_load_metrics/browser/observers/ad_metrics/aggregate_frame_data.h
+++ b/components/page_load_metrics/browser/observers/ad_metrics/aggregate_frame_data.h
@@ -5,6 +5,7 @@
 #ifndef COMPONENTS_PAGE_LOAD_METRICS_BROWSER_OBSERVERS_AD_METRICS_AGGREGATE_FRAME_DATA_H_
 #define COMPONENTS_PAGE_LOAD_METRICS_BROWSER_OBSERVERS_AD_METRICS_AGGREGATE_FRAME_DATA_H_
 
+#include <optional>
 #include <stdint.h>
 
 #include "base/time/time.h"
